Content: Cats Vs Rattlesnake Plains Hunting Patrol

Artyphex opened the pull request:
Plains exclusive, greenleaf exclusive. Your patrol encounters a rattlesnake, if you have high enough stats you kill it, if not, things go poorly.

Artyphex committed:
Cats vs Rattlesnake Patrol

Planes exclusive, greenleaf exclusive. Your patrol encounters a rattlesnake, if you have high enough stats you kill it, if not, things go poorly.

Tiritirimatangi reviewed (commented):
You need to delete the groupmice patrol (which has the old patrol formatting and won't work), and make sure the patrol passes all the checks - currently you're failing the encoding test, which means there's non-ascii characters in your new content?

Artyphex commented:
> You need to delete the groupmice patrol (which has the old patrol formatting and won't work), and make sure the patrol passes all the checks - currently you're failing the encoding test, which means there's non-ascii characters in your new content?

Yup! I'm fixing it now. It's the apostrophes

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/greenleaf.json:
You also need to specify that the leader can't be on the patrol by saying "leader": [0, 0],

Artyphex committed:
Update 1 Content: Cats Vs Rattlesnake Plains Hunting Patrol

Plains exclusive, greenleaf exclusive. Your patrol encounters a rattlesnake, if you have high enough stats you kill it, if not, things go poorly.

Artyphex committed:
Update 2 - The APOSTRAPHES

Windows wants me to suffer

Artyphex committed:
Update 3 Content: Cats Vs Rattlesnake Plains Hunting Patrol

(Hopefully) fixed the apostrophe issue and added a neutral success outcome.

Artyphex committed:
Update 4 Content: Cats Vs Rattlesnake Plains Hunting Patrol

If this doesn't fix that encoding error I'm hurling myself into the sun

Tiritirimatangi reviewed (approved):
Looks like you're good to go! Excellent job!

Artyphex commented:
> Looks like you're good to go! Excellent job!

Woo! Thank you!
